jupyterlab_automl/README.md

```diff
@@ -0,0 +1,61 @@
+# Notebook AutoML Extension
```

Comment:
This can eventually be cleaned up since the setup instructions should now mirror those for the other extensions.